test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -228,6 +232,47 @@ func (cno createNodesOp) patchParams(w *workload) (realOp, error) {
 	return &cno, (&cno).isValid(false)
 }
 
+// createNamespacesOp defines an op for creating namespaces
+type createNamespacesOp struct {
+	// Must be "createNamespaces".
+	Opcode string
+	// Name prefix of the Namespace. The format is "<prefix>-<number>", where number is
+	// between 0 and count-1.
+	Prefix string
+	// Number of namespaces to create. Parameterizable through CountParam.
+	Count int
+	// Template parameter for Count.
+	CountParam string
```

Comment:
Both "Count" and "CountParam" are semantically identical,  possible to just use one instead, maybe if the "CountParam" is not set it could be parsed as "1" for the measured namespace?